assemblyline/common/str_utils.py

```diff
@@ -92,7 +92,10 @@ def escape_str(s, reversible=True, force_str=False):
             return str(s)
         return s
 
-    return escape_str_strict(s.encode('utf-8', errors="backslashreplace"), reversible)
+    try:
+        return escape_str_strict(s.encode('utf-16', 'surrogatepass').decode('utf-16').encode('utf-8'), reversible)
+    except Exception:
```

Comment:
to be more explicit, `except UnicodeDecodeError`

Reply:
There could be tons of possible errors AttributeError, UnicodeDecodeError, SyntaxError... I just want it to try the other on all exceptions.